code/game/turfs/turf.dm

```diff
@@ -79,12 +79,14 @@
 		spawn( 0 )
 			src.Entered(AM)
 			return
-	turfs |= src
```

Comment:
I'm not super familiar with VG code, but this seems problematic.

I'm guessing this is a master turf list that is referenced for `in turfs` instead of `in world`. If this is the case, this means that any turf created after world initialization will not be added to the turfs list...I could...see that being problematic.